bemani/backend/popn/kaimei.py

```diff
@@ -0,0 +1,491 @@
+                # 2: 195
+                # 3: 1915
+                # 4: 1916
+                # 5: 1896
+                # 6: 1908
+                # 7: 1931
+                # 8: 1924
+                # 9: 1925
+                # 10: 1894
+                # 11: 1926
+                # 12: 1927
+                # 13: 1928
+                # 14: 1932, 1933, 1934, 1935
+                # 15: 521
+                # 16: 1936
+                # 17: 1943
+                # 18: 1937
+                # 19: 1938
+                # 20: 1939
+                # 21: 1943
+                # 22: 1941
+                # 23: 1942
+                # 24: 323
+                # 25: 1946
+                # 26: 575
+                # 27: 1947
+                # 28: 1955
+                # 29: 1957
+                # 30: 1958
+                # 31: 1959
+                # 32: 1960
+                # 33: 1961
+                # 34: 1963
+                # 35: 1962
+                # 36: 1968
+                # 37: 1969
+                # 38: 1965
+                # 39: 1970
+                # 40: 1976
+                # 41: 1977
+                # 42: 1978
+                # 43: 1945
+                # 44: 1944
+                # 45: 1999
+                # 46: 2000
+                # 47: 2001
+                # 48: 2002
+                # 49: 2003
+                # 50: 2004
+                # 51: 2005
+                # 52: 267, 1998, 2006
+                # 53: 2011
+                # 54: 2007
+                # 55: 2008
+                # 56: 2009
+                # 57: 2010
+                # 58: 2016
+                # 59: 2012
+                # 60: 2018
+                # 61: 2013
+                # 62: 2015
+                25: mn_tanteisha_phase,
+                # Unknown event (0-3)
+                26: 3,
+                # peace soundtrack hatsubai kinen SP (0-2)
+                # When active, the following songs are available for unlock: 1971, 1972, 1973
+                27: peace_soundtrack,
+                # MZD no kimagure tanteisha joshu (0-2)
+                28: tanteisha_joshu,
+                # Shutchou! pop'n quest Lively (0-5)
+                # When active, the following songs are available for unlock
+                # 1: 1917, 1918
+                # 2: 1919, 1921
+                # 3: 1920, 1922, 1923
+                # 4: 1974
+                29: popn_quest_lively,
+                # Shutchou! pop'n quest Lively II (0-6)
+                # When active, the following songs are available for unlock
+                # 1: 1989, 1990, 1991
+                # 2: 1984, 1985, 1992
+                # 3: 1982, 1983, 1988
+                # 4: 1986, 1987, 1993
+                # 5: 2017
+                30: popn_quest_lively_2,
+            },
+            False,
+        )
+
+    def format_profile(self, userid: UserID, profile: Profile) -> Node:
+        root = PopnMusicModernBase.format_profile(self, userid, profile)
```

Comment:
Technically this should use the magic `super()` instead of hand-specifying the base class, but given there's full traffic tests its trivial for somebody to fix it so I don't think it's a big deal.